api/metric/cumulative.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/api/core"
+)
+
+type Float64Cumulative struct {
+	Instrument
+}
+
+type Int64Cumulative struct {
+	Instrument
+}
+
+type Float64CumulativeHandle struct {
+	Handle
+}
+
+type Int64CumulativeHandle struct {
```

Comment:
What's the difference between a cumulative and handle?

Reply:
For example, the Int64Cumulative contains the metric name and other details. Int64CumulativeHandle associates the metric name with a set of pre-defined label values. I'll add more documentation, sorry.